lib/service/type.rb

```diff
@@ -202,6 +207,21 @@ def reloadable?
     end
 
     private
+
+    def send_signal(sig)
```

Comment:
This name of this method is a bit of a misnomer.  A signal might be sent for a variety of reasons, some of which have nothing to do with killing the process.  This method assumes that the purpose of the signal is to kill the process.  It should be renamed to `kill_with_signal` or `kill_via_signal` or something to indicate its purpose and assumptions.

Reply:
I  have no opinion either way. I've always found the `kill` vs `send` distinction somewhat arbitrary.